scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ForkedJvmTaskExecutionCommandCreator.java

```diff
@@ -198,6 +207,14 @@ private void forwardProActiveProperties(List<String> jvmArguments, PAProperty...
         }
     }
 
+    private void forwardOtherProperties(List<String> jvmArguments, PAProperty... propertiesToForward) {
```

Comment:
Could you add a comment for the function purpose ? the function name is not quite straightforward.

Reply:
Yes, sure.